src/main/groovy/com/marklogic/gradle/task/tests/RoxyTest.groovy

```diff
@@ -0,0 +1,53 @@
+package com.marklogic.gradle.task.tests
+
+import com.marklogic.appdeployer.AppConfig
+import com.marklogic.gradle.task.MarkLogicTask
+import com.marklogic.mgmt.ManageClient
+import com.marklogic.mgmt.ManageConfig
+import com.marklogic.rest.util.Fragment
+import org.gradle.api.tasks.TaskAction
+
+
+class RoxyTest extends MarkLogicTask{
+
+	@TaskAction
+	void runAllTests() {
+		AppConfig config = getAppConfig()
+		String testResultDir = getProject().getProperties().get("mlTestResultDir")
+		String roxyResultDir = "${testResultDir}/roxy-tests"
+		ManageClient testClient = new ManageClient(
```

Comment:
The connection details are interesting here. I'm not sure what the right user to default this to it, but I don't want it to be the admin user, as I want to only use that when the admin role is truly required. I would think that the Roxy tests would use some non-admin user to run them. 

I feel we may need two new properties for this. The problem with using existing properties is that if the client needs to override them, that might break other tasks being run at the same time - e.g. if I run "gradle mlDeploy mlRoxyTest", and I override e.g. mlAdminUsername/mlAdminPassword to be the user I want to run tests with, that might break mlDeploy.

Reply:
Another problem is if the test server requires additional configuration like SSL.